specs/latest/2.0/index.html

```diff
@@ -1092,6 +1095,63 @@ <h2 class="no-toc">Table of contents</h2>
         it is the responsibility of the WebGL API to ensure that data are accessed consistently. This applies
         even if the buffer is currently bound to a transform feedback binding point.
       </dd>
+      <dt>
+        <p class="idl-code">Promise&lt;ArrayBuffer&gt; getBufferSubDataAsync(GLenum target, GLintptr offset, ArrayBuffer returnedData, unsigned long returnedOffset = 0, unsigned long returnedSize = 0)
+        </p>
+      </dt>
+      <dd>
+        Let <code>buffer</code> be the WebGLBuffer bound to the passed
+        target. Read <code>numBytes</code> bytes from <code>buffer</code> starting at byte
+        offset <code>offset</code> and write them to <code>returnedData</code> starting at byte
+        offset <code>returnedOffset</code>. <code>numBytes</code> is computed via the following
+        algorithm:
+
+        <ul>
+        <li> If <code>returnedSize</code> is zero, <code>numBytes = returnedData.byteLength - returnedOffset</code></li>
+        <li> Otherwise, <code>numBytes = returnedSize</code>.
+        </ul>
+
+        This command executes serially with respect to others in the command stream, but the data is
+        only written to <code>returnedData</code> when the promise is fulfilled; it is available in
+        the Promise's success callback, but not earlier.
+
```

Comment:
Should we specify if the data should be when the call is made, or when the promis is fulfilled, or anything in between? Since it's an async call, after this call, we can modify the content of the buffer.